### ISSUE TITLE
IT Unstable case `TestSumAvg`

### ISSUE THREAD

**Issue by aytrack** [pingcap/tidb#30297]
## Bug Report

Please answer these questions before submitting your issue. Thanks!
in ci https://ci.pingcap.net/blue/organizations/jenkins/tidb_ghpr_coverage/detail/tidb_ghpr_coverage/1416/pipeline/
```bash
2021-12-01T02:26:54.531Z] --- FAIL: TestSumAvg (1.39s)
[2021-12-01T02:26:54.531Z]     dbtestkit.go:81: 
[2021-12-01T02:26:54.531Z]         	Error Trace:	dbtestkit.go:81
[2021-12-01T02:26:54.531Z]         	            				server_test.go:1864
[2021-12-01T02:26:54.531Z]         	            				server_test.go:117
[2021-12-01T02:26:54.531Z]         	            				server_test.go:1853
[2021-12-01T02:26:54.531Z]         	            				tidb_test.go:1114
[2021-12-01T02:26:54.531Z]         	Error:      	Received unexpected error:
[2021-12-01T02:26:54.531Z]         	            	No Pem data found, data: #28000Access denied for user 'root'@'127.0.0.1' (using password: YES)
[2021-12-01T02:26:54.531Z]         	Test:       	TestSumAvg
[2021-12-01T02:26:54.531Z]         	Messages:   	sql:select avg(a), avg(b), avg(c) from sumavg, args:[]
```

**Comment by hawkingrei** [comment]
Closed by #30306

**Comment by hawkingrei** [comment]
we can re-open it if the error happens again.
